media/css/firefox/new/compare.scss

```diff
@@ -0,0 +1,487 @@
+
+    &:after {
+        background: url('/media/img/firefox/new/compare/illo-good.png') center bottom no-repeat;
+        background-size: 386px 165px;
+        bottom: 0;
+        content: '';
+        display: block;
+        height: 165px;
+        left: 50%;
+        margin-left: -193px;
+        position: absolute;
+        width: 386px;
+    }
+}
+
+@media #{$mq-sm} {
+    .section-good {
+        padding-bottom: 230px;
+
+        &:after {
+            background-size: 579px 247px;
+            height: 247px;
+            margin-left: -289px;
+            width: 579px;
+        }
+    }
+}
+
+@media #{$mq-md} {
+    .section-good {
+        padding-bottom: 40px;
+
+        &:after {
+            @include at2x('/media/img/firefox/new/compare/illo-good.png', 773px, 331px);
+            height: 331px;
+            left: auto;
+            margin-left: 0;
+            position: absolute;
+            right: 35%;
+            width: 773px;
+        }
+    }
+}
+
+@media #{$mq-xl} {
+    .section-good:after {
+        right: auto;
+        left: 35%;
+        margin-left: -386px;
+    }
+}
+
+//* ====================================================================== */
+//  Buttons
+.compare {
+    .mzp-c-button.mzp-t-download {
+        @include gradient-vertical($start-color: $color-green-50, $end-color: $color-green-70);
+        background-color: $color-green-70;
+        border-radius: 3px;
+        border: none;
+        box-shadow: none;
+        color: $color-white;
+        line-height: 1.125;
+        padding: ($padding-md + 2px) ($padding-lg + 2px); //2px extra padding to compensate for no border.
+
+        &:hover,
+        &:focus {
+            @include gradient-vertical($start-color: $color-green-60, $end-color: $color-green-80);
+            background-color: $color-green-80;
+            box-shadow: 4px 4px 0 0 $color-black;
+            color: $color-white;
+            transition: box-shadow 100ms;
+        }
+
+        &:active {
+            box-shadow: 2px 2px 0 0 $color-black;
+            transition: box-shadow 0ms;
+        }
+    }
+}
+
+
+//* ====================================================================== */
+//  Tables
+
+$image-width: 21px;
+$image-top-height: 19px;
+$image-bottom-height: 28px;
+
+.c-compare-table {
+    width: 100%;
+    max-width: 516px;
+    margin: ($layout-md - $image-top-height) auto ($layout-xl - $image-bottom-height) auto;
+
+    th {
+        font-weight: normal;
+        text-align: left;
+    }
+
+    th,
+    td {
+        padding: $spacing-sm;
+    }
+
+    td {
+        text-align: center;
+    }
+
+    thead th {
+        text-align: center;
+    }
+
+    tbody {
+        tr {
+            background-color: #fff;
+
+            &:nth-child(odd) {
+                background-color: #f7f7f7;
+            }
+        }
+
+        td {
+            width: 80px;
+
+            span {
+                display: block;
+            }
+        }
+    }
+}
+
+@media (max-width:479) {
```

Comment:
Missing units, I don't think this MQ is being fired.

Reply:
And that reminds me, we should have the breakpoints as variables in Protocol so we can write custom MQs if we need to. (It doesn't make sense to define the grid at the small mobile sizes and then over-ride everything back to the default behaviour at large ones. Though we can do that if we decide to take a hard line on acceptable MQs.)

Reply:
You wrote this 😉 Presumably a dev artifact, but I failed to notice it. Since it isn't taking effect anyway we can just remove it.

Reply:
I don't know when it broke but it was tested before I handed it off (I probably was the one who broke it). It improves the mobile experience though (I think) so should stay.

Reply:
We do have breakpoints as tokens, it's just poorly documented and hard to reference: https://github.com/mozilla/protocol-tokens/blob/master/tokens/_aliases.yml#L135 A token reference is on the to-do list.

So this can be rewritten as `@media (max-width: $screen-sm)`